zrml/prediction-markets/src/tests.rs

```diff
@@ -1817,7 +1817,7 @@ fn it_allows_to_sell_a_complete_set() {
         // Check the outcome balances
         let assets = PredictionMarkets::outcome_assets(0, &market);
         for asset in assets.iter() {
-            let bal = Tokens::free_balance(*asset, &BOB);
+            let bal = AssetManager::free_balance(*asset, &BOB);
```

Comment:
Why not using the router here, since outcome assets are assumed to be handled by pallet_assets?

Reply:
Consequently asked for every instance in this file handling outcome assets.

Reply:
`AssetManager` internally uses `AssetRouter` for anything that is not ZTG.